scripts/lindtool.sh

```diff
@@ -252,13 +252,13 @@ case $1 in
     compile_wasmtime|cpwasm)
         echo -e "${GREEN}$compile_wasmtime_cmd${RESET}"
         if [ "$pmode" -eq 0 ]; then
-            eval "$compile_wasmtime_cmd"
+            bazel build //:make_wasmtime
```

Comment:
From user perspective, nothing changes here, correct? Their experience would be unchanged?